app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -11,41 +11,69 @@ import com.jerboa.db.dao.AccountDao
 import com.jerboa.db.dao.AppSettingsDao
 import com.jerboa.db.entity.Account
 import com.jerboa.db.entity.AppSettings
-import com.jerboa.feat.SwipeToActionPreset
+import com.jerboa.toBool
 import com.jerboa.ui.theme.DEFAULT_FONT_SIZE
 import java.util.concurrent.Executors
 
+// Warning: Be careful about changing any of these defaults, as it will mean you'll need to
+// regenerate the database
+// Unfortunately can't use enum ordinals here, because Room's compile time annotations
+// don't allow it
+//
+// Use Int for Bools
+const val DEFAULT_THEME = 0
```

Comment:
I really wanted to be able to use the Enums and ordinals here, but Room makes you use constants for their `currentValue` annotations.